backend/src/test/java/gov/cdc/usds/simplereport/validators/FileValidatorTest.java

```diff
@@ -391,6 +380,42 @@ void testResultsFile_invalidValues() {
     indices.forEach(i -> assertThat(i).isEqualTo(List.of(2)));
   }
 
+  void testResults_invalidFile(String fileName, String errorMessage) {
+    // GIVEN
+    InputStream input = loadCsv(fileName);
+
+    // WHEN
+    List<FeedbackMessage> errors = testResultFileValidator.validate(input);
+
+    /// THEN
+    assertThat(errors).hasSize(1);
+
+    List<String> errorMessages = errors.stream().map(FeedbackMessage::getMessage).toList();
+
+    assertThat(errorMessages).contains(errorMessage);
+  }
+
+  @Test
```

Comment:
Sonar was complaining that these tests were too similar and wanted us to parameterize them, so abstracted them up here